packages/flutter/lib/src/cassowary/param.dart

```diff
@@ -6,45 +6,66 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'term.dart';
 
+/// Represents a variable inside the layout solver.
 class Variable {
   static int _total = 0;
 
+  /// Create a new variable with the given constant value.
   Variable(this.value) : _tick = _total++;
 
   final int _tick;
 
+  /// The current value of the variable.
   double value;
 
+  /// An optional name given to the variable. This is useful in debugging
+  /// solver state.
   String name;
 
+  /// Variables represent state inside the solver. This state is usually of
+  /// interest to some entity outside the solver. Such entities can (optionally)
+  /// associate themselves with these variables. This means that when solver
+  /// is flushed, it is easy to obtain a reference to the entity the variable
+  /// is associated with.
   Param get owner => _owner;
   Param _owner;
 
+  /// Used by the solver to apply updates to this variable. Only updated
+  /// variables show up in solver flush results.
   bool applyUpdate(double updated) {
```

Comment:
Reword to:
`Applies updates to this variable. Used by the solver. Only updated variables show up in solver flush results.`

Add clarifications saying what it is that that is updated.